src/sdk/evaluation.ts

```diff
@@ -210,25 +224,22 @@ async function windupEvaluation(state: EvaluationState): Promise<void> {
 
 async function evaluate(
     config: EvaluationConfig
-): Promise<{
-    run_id: string;
-    dataset_id: string | undefined;
-    session_ids: string[];
-    status: Status;
-}> {
+): Promise<EvaluationResult> {
     validateRequirements(config);
 
+    const suite = config.suite || "default";
```

Comment:
In Python, if `suite` is not passed, it tries to get the folder name to be used before falling back to `default`. I didn't replicate this behavior here because this can be more complex and potentially unreliable.